src/client/components/ActivityFeed/activities/DataHubEvent.jsx

```diff
@@ -13,6 +13,9 @@ export default function DataHubEvent({ activity: event }) {
   const eventObject = event.object
   const name = eventObject.name
```

Comment:
```suggestion
  const eventName = eventObject.name
```

Maybe change this line just so there's no confusion with the `.name` on lines 16-18!